chasemapper/config.py

```diff
@@ -54,6 +54,9 @@
     'bearing_color': 'black',
     'bearing_custom_color': '#FF0000',
 
+    # History 
+    'reload_last_position': True
```

Comment:
Make this False by default... I certainly wouldn't want this on all the time.

Reply:
Done